easybuild/easyblocks/b/boost.py

```diff
@@ -121,9 +121,36 @@ def configure_step(self):
             # http://www.boost.org/doc/libs/1_47_0/doc/html/mpi/getting_started.html
             # let Boost.Build know to look here for the config file
             f = open('user-config.jam', 'a')
-            f.write("using mpi : %s ;" % os.getenv("MPICXX"))
-            f.close()
+ 
+            # Check if using a Cray toolchain and configure MPI accordingly
+            if (self.toolchain.name.startswith('Cray') and 
+                self.toolchain.PRGENV_MODULE_NAME_SUFFIX in ['gnu', 'intel', 'cray']):
+                self.log.info("PRGENV_MODULE_NAME_SUFFIX is %s " % (self.toolchain.PRGENV_MODULE_NAME_SUFFIX))
+                craympichdir=os.getenv('CRAY_MPICH2_DIR')
+                craygccversion=os.getenv('GCC_VERSION')
```

Comment:
If you're not with gnu you should update the easyblock :) 

I fixed that so that it is clear that only gnu is supported for now